lib/led/rgb.js

```diff
@@ -269,7 +269,7 @@ RGB.prototype.color = function(red, green, blue) {
     // not a reference to the state object itself.
     colors = this.isOn ? state : state.prev;
     return RGB.colors.reduce(function(current, color) {
-      return (current[color] = colors[color], current);
+      return (current[color] = Math.round(colors[color]), current);
```

Comment:
If it's floor, then it's likely that many colors won't be able to reach 0xff at 100% intensity. If it's ceiling, it should be fine, since anything at 0% intensity will reach 0x00. I figured rounding was better than ceiling, since a value like 32.1 should end up as 32 instead of 33.